### PR TITLE
Add gamemode where the point is to survive as long as possible while dodging some angry bots

### PR DESCRIPTION
**Pull request by bjornenalfa**

### CODE REVIEW

**Comment by joelnir** [comment]
I check this

File: src/game/entities/Dangerbot.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    this.collision = new CollisionCircle(radius);
+    this.collision.setEntity(this);
+    this.collision.addListener((dangerbot, victim) => {
+      if (victim.isPlayer()) {
+        victim.die();
+      }
+    });
+  }
+
+  /* eslint-disable class-methods-use-this */
```
Comment (joelnir) [inline src/game/entities/Dangerbot.js]: Remove this

File: src/game/gamemodes/Dodgebot.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+import HighscoreList from '../HighscoreList';
+
+// Respawn time in seconds
+const RESPAWN_TIME = 1;
+
+// How many walls the arena should have.
+const WALLS = 4;
+
+/*
+  Knock off gamemode, get score by knocking other players off the arena.
```
Comment (joelnir) [inline src/game/gamemodes/Dodgebot.js]: Wrong descriptions

**Comment by bjornenalfa** [comment]
Everything should be fixed now.

**Comment by joelnir** [comment]
The time in the highscore list looks strange. Otherwise everything seems good. Really fun gamemode.

**Pull request merged**